### PR TITLE
Update deprecated set-env

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

File: charts/renku/requirements.yaml [hunk 1]
```diff
@@ -29,6 +29,6 @@ dependencies:
   repository: "https://charts.bitnami.com/bitnami"
   condition: postgresql.enabled
 - name: keycloak
-  version: 4.10.1
-  repository: "@stable"
+  version: 4.10.2
```
Comment (rokroskar) [inline charts/renku/requirements.yaml:32]: why the change in this PR?

File: .github/workflows/pull-request-test.yml [hunk 2]
```diff
@@ -94,6 +94,7 @@ jobs:
       run: |
         echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
         helm repo add stable 	https://kubernetes-charts.storage.googleapis.com
+        helm repo add codecentric https://codecentric.github.io/helm-charts
```
Comment (rokroskar) [inline .github/workflows/pull-request-test.yml:97]: This should be added to instructions that describe deploying renku from source - but I'm not sure it belongs in this PR?

**Review by rokroskar — CHANGES_REQUESTED** [review]
Thanks @pameladelgado! I don't think we should be making changes to the keycloak version here - if it's inevitable then I suggest we do the change here but open an issue for adapting the docs.

File: charts/renku/requirements.yaml (hunk 1 above)
Comment (pameladelgado) [reply charts/renku/requirements.yaml:32]: > why the change in this PR?

Because in the new codecentric repo that's the oldest chart available. The deployment does not work otherwise, so it is indeed a small inevitable update. Luckily, this is just a bugfix change in versions so it should not break things. Authorization seem to work just fine in that deployment.
It seems that we might want to work on #1003 sooner rather than later.

File: .github/workflows/pull-request-test.yml (hunk 2 above)
Comment (pameladelgado) [reply .github/workflows/pull-request-test.yml:97]: Yes, documentation should be updated.
I also did not included it here because I was not sure it belongs in this PR but I can certainly add it. Bitnami repo should also be added to the docs.

File: charts/renku/requirements.yaml (hunk 1 above)
Comment (lorenzo-cavazzi) [reply charts/renku/requirements.yaml:32]: I was going to push the very same changes cause I had the issue with `keycloack` :smile: 

Some extra references for the reviewers:
* Deprecated: https://github.com/helm/charts
* New source: https://github.com/codecentric/helm-charts
* Problem with the version:
```
helm search repo keycloak --versions
NAME            CHART VERSION   APP VERSION     DESCRIPTION                                       
stable/keycloak 4.10.1          5.0.0           DEPRECATED - Open Source Identity and Access Ma...
stable/keycloak 4.10.0          5.0.0           Open Source Identity and Access Management For ...
```

File: .github/workflows/pull-request-test.yml (hunk 2 above)
Comment (rokroskar) [reply .github/workflows/pull-request-test.yml:97]: sure that's fine - please make an issue for it. Thanks!

**Review by rokroskar — APPROVED** [review]
